pages/dkp/kommander/2.1/backup-and-restore/index.md

```diff
@@ -73,6 +76,31 @@ data:
         #   aws_secret_access_key=<REDACTED>
 ```
 
+After you have configured the `ConfigMap` with **MinIO**, OR **Amazon S3**, add the `configOverrides` value to the Velero `AppDeployment`. This applies the `ConfigMap` to your instance after the cluster is configured:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: apps.kommander.d2iq.io/v1alpha2
+kind: AppDeployment
+metadata:
+  creationTimestamp: "2022-06-10T04:46:31Z"
```

Comment:
@mhrabovcin should we edit these type of values or delete this line? Not sure, but the time stamp makes me believe this is an environment-specific value? The same with lines 93, 94, 98?